src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+public abstract class Figure implements Drawable {
+    protected Color color;
+    protected double area;
```

Comment:
Think about which variables should be class-level. The 'area' might not need to be a field in the 'Figure' class if it's calculated differently for each figure type and not shared across different methods.